script/benchmark

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+
+set -e
+
+function benchmark() {(
+  cd tmp/bench-site
+  bundle update
+  time bundle exec jekyll build
+)}
+
+echo "travis_fold:start:setup"
+rm -rf tmp/benchmark tmp/bench-site
+mkdir -p tmp/bench-site
+git clone --depth 1 https://github.com/jekyll/benchmarking.git tmp/benchmark
+echo "travis_fold:end:setup"
+
+echo "travis_fold:start:generate_site.sh"
+tmp/benchmark/generate_site.sh 1000 tmp/bench-site
```

Comment:
I'm wondering if instead of a temp site we should measure the time of the docs site to have a more realistic example?

Reply:
One problem with the docs site is that it's pretty small and always generates pretty quickly.
We need something that takes more than a few seconds to generate so we can really see the difference in build times.

Reply:
Good point, makes total sense then 👍